hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AuthenticationBaseMessageTask.java

```diff
@@ -314,21 +286,67 @@ public Permission getRequiredPermission() {
         return null;
     }
 
-    private class OperationInfo {
-        private final Member member;
-        private final Future future;
+    private final class MultiTargetCallback {
```

Comment:
It is pretty confusing that MultiTargetCallback is not a callback implementation at all.